cmd/imagec/imagec.go

```diff
@@ -379,6 +382,17 @@ func WriteImageBlobs(images []*ImageWithMeta) error {
 	if err := os.RemoveAll(destination); err != nil {
 		return fmt.Errorf("Failed to remove download directory: %s", err)
 	}
+
+	if len(images) > 0 && options.imagefile != "" {
```

Comment:
I've talked with @cgtexmex and I've decided that this will not support multiple tags per image until his changes go in. Once they're in, I can refactor this to take advantage of them.